Ping atk if this needs updating.

This prevents vscode from fetching prebuilt ripgrep from Microsoft
during build, which unbreaks build on platforms where MS deos not
ship a prebuilt ripgrep.

diff --git a/package.json b/package.json
index 5d6022a2a40..f66ee44eef0 100644
--- a/package.json
+++ b/package.json
@@ -63,7 +63,7 @@
     "@parcel/watcher": "2.0.5",
     "@vscode/debugprotocol": "1.51.0",
     "@vscode/iconv-lite-umd": "0.7.0",
-    "@vscode/ripgrep": "^1.14.1",
+    "@vscode/ripgrep": "https://github.com/atk/void-vscode-ripgrep.git",
     "@vscode/sqlite3": "4.0.12",
     "@vscode/sudo-prompt": "9.3.1",
     "@vscode/vscode-languagedetection": "1.0.21",
diff --git a/remote/package.json b/remote/package.json
index 4bc40998093..9d22d408c02 100644
--- a/remote/package.json
+++ b/remote/package.json
@@ -6,7 +6,7 @@
     "@microsoft/applicationinsights-web": "^2.6.4",
     "@parcel/watcher": "2.0.5",
     "@vscode/iconv-lite-umd": "0.7.0",
-    "@vscode/ripgrep": "^1.14.1",
+    "@vscode/ripgrep": "https://github.com/atk/void-vscode-ripgrep.git",
     "@vscode/vscode-languagedetection": "1.0.21",
     "applicationinsights": "1.4.2",
     "cookie": "^0.4.0",
